packages/js/src/target/node/worker.js

```
@@ -0,0 +1,22 @@
+import { Worker } from "node:worker_threads";
+import { fileURLToPath } from "node:url";
+import { dirname, join, resolve } from "node:path";
```

ESLint

[error] 'join' is defined but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)